src/modules/ToDoList/ToDoTask.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import '../../styles/ToDoListStyles/ToDoList.css'
+import '../../styles/ToDoListStyles/ToDoTask.css'
+import {completeTask, deleteTask, modifyTaskContent} from "./_actionsDefinitions";
+import {connect} from "react-redux";
+import {isTaskComplete, getTaskByID} from './_selectors'
+
+
+class ToDoTask extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {modifying: false}
+    }
+
+    storeChangingInput = (event) => {
+        this.props.updateTask(this.props.id, event.target.value);
```

Comment:
don't call the action on input change, store the changes in the state but dispatch action only on user request.